docs-2.0/3.ngql-guide/15.text-based-index-statements/4.search-with-fulltext-index.md

```diff
@@ -0,0 +1,59 @@
+# Full-text search
+
+```ngql
+LOOKUP ON {<tag> | <edge_type>} WHERE <expression> [YIELD <return_list>]
+
+<expression>
+    PREFIX | WILDCARD | REGEXP |FUZZY
```

Comment:
add a sentence and a example for these four commands

Reply:
I did